manim/utils/config_utils.py

```diff
@@ -314,6 +322,25 @@ def _parse_cli(arg_list, input=True):
         "--config_file", help="Specify the configuration file",
     )
 
+    parser.add_argument(
+        "-q", "--quiet",
+        action="store_true",
+        help="Quiet mode. Equivalent to -v CRITICAL. It can be overridden by -v and -L",
+    )
+    parser.add_argument(
+        "-v", "--verbose",
+        type=str,
+        help="Verbosity level. Also changes the ffmpeg log level unless the latter is specified",
+        metavar="loglevel",
+        choices=constants.VERBOSE_CHOICES,
+    )
+    parser.add_argument(
+        "-L", "--ffmpeg",
+        type=int,
+        help="Log level for ffmpeg",
+        metavar="loglevel",
+        choices=[-8, 0, 8, 16, 24, 32, 40, 48, 56],
```

Comment:
**Suggestion (non-blocking)** I think this should use `VERBOSE_FFMPEG_MAP` directly instead of asking the user to input verbose FFmpeg int, for convenience.

Reply:
I think it is normal that `--fmpeg 0`, `--fmpeg -8`, `--fmpeg 16` produce the same output because they all correspond to low value of verbosity. More specifically -8 does not output anything, 0 seems equivalent to -8 by reading the doc but may be used in the future, 8 shows only fatal errors, and 16 shows all errors.

I can totally add a flag to disable the progessbar to give more control overriding the verbose option in that sense.

I agree with you it would be more readable to use a string for the ffmpeg but actually I see now it can be a string or the equivalent int so I'll change that.

Reply:
> and 16 shows all errors.

So why is there choices greater than 16 ? Sorry if I'm bothering you with my question but i really don't get it lol